ci-setups/mr-title-validation.md

```diff
@@ -0,0 +1,90 @@
+
+# Merge Request Validation CI/CD Setup on Gitlab
+
+## Table of Contents
```

Comment:
For the table of contents, let's generate it using `yzhang.markdown-all-in-one` with proper headings and subheadings

Reply:
Done, made the change.